docs/services/matrix.md

```diff
@@ -85,5 +85,29 @@ By default, a SQL-Lite database is used, so we still need to comment it out.
 
 ```
 
-Now we are finish and we can start the container with `docker-compose up -d`
+Now you are finish and you can start the container with `docker-compose up -d`
 
+### Reset password of user 
+
+If you want to reset the assword run 
+```yaml 
+  docker exec -it -u www-data matrix hash_password -p PASSWORD
```

Comment:
- remove spaces, in front of code block

- change to docker-compose command to be able to use the "matrix" service defined in the docker compose. If you want to use the docker context, you need to get the container name which is in the following format by default: `project_serviceName_1` (`project` is the directory in which the `docker-compose.yml` is located, `serviceName` is the name of the service (in this case `matrix`)):
`docker-compose exec -u www-data matrix hash_password -p PASSWORD`

Is it furthermore possible, to read the password from stdin? If you execute this command the password will appear in the bash_history file (at least unless you didn't execute `set +o history`).